final/src/components/timesheets/TimesheetRow.js

```diff
@@ -1,47 +1,56 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
-import {Button} from 'react-bootstrap';
+import { Button } from 'react-bootstrap';
 import { withRouter } from 'react-router';
 
 class TimesheetRow extends Component {
-
   handleClick(timesheet) {
-    if(timesheet.deleted){
+    if (timesheet.deleted) {
       timesheet.deleted = false;
       this.props.actions.restoreTimesheet(timesheet).then(this.props.actions.listTimesheets);
-    }
-    else{
+    } else {
       timesheet.deleted = true;
       this.props.actions.removeTimesheet(timesheet).then(this.props.actions.listTimesheets);
     }
   }
 
   showDetail(timesheet) {
-    if(timesheet.deleted) {
+    if (timesheet.deleted) {
       console.log('You cannot edit a deleted timesheet.');
       return;
     }
 
-    this.props.history.push('/employees/' + timesheet.user_id + '/timesheets/detail/' + timesheet._id);
+    this.props.history.push(
+      '/employees/' + timesheet.user_id + '/timesheets/detail/' + timesheet._id
```

Comment:
I guess prettier doesn't automatically convert to template strings eh? ;-)

Reply:
ugh I wish. That'd probably be a fairly brittle thing to update I'd guess tho?